audit_log/tests/test_utils.py

```diff
@@ -61,3 +72,148 @@ def test_get_response_status(status_code, audit_status):
     res_mock = Mock(status_code=status_code)
 
     assert get_response_status(res_mock) == audit_status
+
+
+@pytest.mark.parametrize(
+    "old_dict, new_dict, expected_diff",
+    [
+        ({}, {}, {}),
+        ({"a": 1}, {"a": 1}, {}),
+        ({"a": 1}, {"a": 2}, {"a": 2}),
+        ({"a": 1}, {"b": 2}, {"b": 2}),
```

Comment:
IMHO e.g. adding a note to the `diff_dicts` function's docstring that it won't return deleted keys in the output at all would make this behavior even more explicit beyond the current "Returns a new dictionary containing only the changed keys and values.".

Reply:
A note added